nilearn/plotting/img_plotting.py

```diff
@@ -587,7 +613,15 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
                     np.logical_not(stat_map_data._mask)])
         stat_map_max = np.nanmax(stat_map_data)
         stat_map_min = np.nanmin(stat_map_data)
+
+    if symmetric_cbar == 'auto':
+        symmetric_cbar = stat_map_min < 0 and stat_map_max > 0
```

Comment:
I prefer to enclose boolean expression in parenthesis `
symmetric_cbar = (stat_map_min < 0 and stat_map_max > 0)`